public/js/main.js

```diff
@@ -0,0 +1,25 @@
+
+$("#newGame").click(function(){
+	$("td").removeClass()});
```

Comment:
Also, it reputedly we use two spaces instead of tabs. Would be a great habit to start getting in to!

```
$("#newGame").click(function(){
  $("td").removeClass()
});
```